src/styles/components/application-preloader/styles.less

```diff
@@ -53,7 +53,8 @@
 
   .application-preloader-spinner {
     animation: application-preloader-spinner 1s linear infinite;
-    background-image: url('../../../img/components/application-preloader/application-preloader-spinner.png');
+    // from file '../../../img/components/application-preloader/application-preloader-spinner.png'
+    background-image: url('data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAGQAAABkCAYAAABw4pVUAAAGDklEQVR4AezbBW8iQRiAYepo3d0VWdzh3N3d7/7/P+i9TT6SL5N6l6kNyXMu07zAjrCBi/7Y3d3twSCmsQYPRVTQwC3cE7fRRA1lZLCJWQyhN+AeJw7Qh0lso4o7yl1xz3BfPFAeikfKYwnoSaTg/qNwEcJYRgm3lNviiCgS5IgY4onhFrYQve4RejCLLG7gprAZ5Kl4JhpYRu91e0taQ11CiP2DyK+VkMQKpjCIGELoRafoRVh+bwgz2EAGDYm0bxDlufxaEqGrHCKEdQnRFGaQliyWMYROH8fQhVGJVMWT/YKIF/J9GtGrFKIbK2go+wXxMIluy2ObQ0XHEC+Ul0ii57LHGEMJdQlhRilhHsEL8gpewz0zhvIIc5cxRBAJ1ETdiFLAFDov4Ng7sYC7RpBXSgORyxJjBCUJYQbJYxwdl+Dr6MAs7uOV4TWeYeYifwGdWEJV1JQqFm2+InyeCGzjhYrR8gapC/d1yXQziYrQUeIIXYHJSRQ1HUO5heBFul5kUFGq8v30FZy+r+0T5C0eIXreg4sgh7IRJIvYFV5TDeORitHyDEPnNah+5FFCWUXZtrWWuABv03UV4514hTH7rwyJoZSxansGdQEmMgUJob3GkO1rRsmwcI1imNPjlIrxXjxH1MYOrYeiKImZgDtC2MJ75QOeINjOl+cOCqIoFgKth4viqRgt99qyTpG3pIJhBR0B99BvXyUjyEdk/P6PhpA3Ymy4GAe+kzRVjJZZPw+UMsgr3qFTWxelF88kxCfxxpcNSXkl5ERevj969uCijOgg4rYfK9KcYfLY/4CLsmkE+YzFs+xyJo0Y6yf+h1yUW/isvD7VyaMcbWY19zmmU28xfZQYX0TuVBdykRWn37l1UVL4onxG7KRrjoySONPixgXpwgsjSvkkU7aMYfjMo3JRFiXEV/EZkeNeO9Iigx23APRtFf9CBfmGwnE+p+QhrYz4NioXZRXflM8IHvU5Kk+J+/7qcNsqr40om0etyj1lwvdRuShxFeM7Hh928ORp7gaXtp22fjcM7PcHp5BSVts2KhflvoT4IbL7v13pIG6q2+6L+w/l1X5Hs0lDV1tH5W5W+m5ECZsHUCqGhbcrF+URfior+jdndRArW+wuSMYIUjf37RMtVg6gXJAp/FLe6dV5XMWIW9tIdBuO340ofa1bkuPKmrVRuSgvjSATrQt6XJm3Oip3mvhbWQ/sVcGOCmJvu8QFyRpB8gG56XJHGbQ2IhdkDX+UuwG5DW1bBQlbG5ELMmkEebb3iyvYVoLWRuSCDBtB3sjLRgWxucPrgsTwV/m494vr2MK29bufXJCgEeR7QGIolk8I3eLwn/Lnf3tngeNQDMTQMjMzw/1PuOgvWbOMbvtd6R2g85QMNE0wNiHkQmzpxEL0Y3cL2bMQcVL3B33ImSg7KlohaxbixlAvZA4RJ9ByVPSXjJ2IgaOiFdJmIf7bweVcTnkEa0dF3y0eicNl/ITr0vdAuNISCxkHIX1HRT8GZiELR0UrpAARe0KcRyxlFYQ0HRX9VRA7QtyPWEgJIpiCI6OVMicZWx8JuowxypZY+RdEfde+ISEbJ/fLSO4bYqFfJe5JNoG6I6OVMoCINZjrV4n/lLgmVq649FJ6JAOo32DyvRyLIGTkyGil1EnGErgMFksZJTLAwofp9GXwIpEBxC+sWUoVIpiBS2GtlA7JmIOuVorvDBwlMsAMdz35Ixw+TkgGcOWlTvITFoIk3/L2pR2tTBMZRMdStA/VT0jGBPS0JbFXyhgimKHnXtqcMiQZYzDykVRt9dVnGUTHp1d0fUrzFSFDUHfC1yX74StCBkj45Wt54P7WtrAuywh0ca1E9hJFJNzqXR79IKMPesgvVXWZHETkwG1KwZdrvCaE6CL/lP4zCCQgD3JSIYJtrBFlgC7ogCZWVwFB+QsBBYKEKKToxdRfk0G0QQs8iqyiICiGAGYDOcCBL4ISKIJ3pchlCLayCgL+rhDQBA2iDmqgCipEGZQ+KQSrwo1lFYFXCMm/L8HHj0oIcPOPhBR+tiV5ApB/oIjAVklApBolUM65CgH3I73MdzrpM08AAAAASUVORK5CYII=');
```

Comment:
We should use the webpack url-loader instead of manually converting this one into a base64 string, to make sure that we always display the correct spinner and don't forget to update this one when we get new assets.